src/test/java/be/seeseemelk/mockbukkit/ChunkSnapshotMockTest.java

```diff
@@ -29,7 +29,7 @@ class ChunkSnapshotMockTest
 	void setUp()
 	{
 		MockBukkit.mock();
-		world = new WorldMock(Material.GRASS, 0, 319, 4);
+		world = new WorldMock(Material.SHORT_GRASS, 0, 319, 4);
```

Comment:
Why `SHORT_GRASS` here? We should either use `GRASS_BLOCK` or something different entirely.